sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1515,11 +1528,43 @@ def process(self, element, *schema_side_inputs):
 
     if not self.with_batched_input:
       row_and_insert_id = element[1]
+      row_byte_size = get_deep_size(row_and_insert_id)
+      # maintain buffer byte size for each destination
+      self._destination_buffer_byte_size.setdefault(destination, 0)
+      # send large rows that exceed BigQuery insert limits to DLQ
+      if row_byte_size >= self.max_insert_payload_size:
+        row_mb_size = row_byte_size / 1_000_000
+        max_mb_size = self.max_insert_payload_size / 1_000_000
```

Comment:
this indicates max_insert_payload_size must be greater than 1_000_000. We should validate this when calling init.

Reply:
This would be true if we are using double slash, which acts as a `math.floor()` after division. But here we are only using one slash for division here which will result in a float when necessary (e.g. 500 / 1_000_000 = 0.0005).

Reply:
I see. Forgot that difference.